proxy/proxy.go

```diff
@@ -689,6 +689,7 @@ func (p *Proxy) makeBackendRequest(ctx *context) (*http.Response, *proxyError) {
 	if req.Header.Get("Connection") == "Keep-Alive" {
 		req.Header.Set("Connection", "keep-alive")
 	}
+	p.metrics.IncCounter("downstream." + req.Proto)
```

Comment:
upstream and downstream may be ambiguous. Can we find a different key? One suggestion: request.incoming and request.outgoing?

Reply:
ok